src/VisualStudio/Core/Def/PublicAPI.txt

```diff
@@ -1,6 +1,8 @@
 Microsoft.VisualStudio.LanguageServices.Progression.GraphNodeCreation
 Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace
 Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace.CreatePortableExecutableReference(string filePath, Microsoft.CodeAnalysis.MetadataReferenceProperties properties) -> Microsoft.CodeAnalysis.PortableExecutableReference
+Microsoft.VisualStudio.Shell.Interop.IComWrapperFactory
+Microsoft.VisualStudio.Shell.Interop.IComWrapperFactory.CreateAggregatedObject(object managedObject) -> object
```

Comment:
Actually, you're right. It can be made internal since we only consume it and don't implement it.